x-pack/plugins/ml/server/models/data_recognizer/modules/auditbeat_process_docker_ecs/ml/docker_rare_process_activity_ecs.json

```diff
@@ -0,0 +1,40 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Auditbeat: Detect rare process executions in docker containers (ECS)",
+  "groups": ["auditd"],
+  "analysis_config": {
+    "bucket_span": "10m",
+    "detectors": [
+      {
+        "function": "rare",
+        "by_field_name": "process.executable",
+        "partition_field_name": "container.name"
```

Comment:
I think we should give this a user-friendly detector description. `rare process execution` maybe?